tests/ci/integration/run_openssh_integration.sh

```diff
@@ -6,9 +6,9 @@ source tests/ci/common_posix_setup.sh
 
 # Set up environment.
 
-# ROOT
+# SYS_ROOT
 #  |
-#  - AWS_LC_DIR
+#  - SRC_ROOT
 #    |
 #    - aws-lc
```

Comment:
The documentation makes it look like "aws-lc" is a directory under `$SRC_ROOT`, but "aws-lc" is the `$SRC_ROOT`.

Reply:
Fixed :)